docs/Catalog/SearchQuery.md

```diff
@@ -47,38 +47,40 @@ The following are all valid search parameters:
 
 **Fields**
 
-- `comment`: Package comment. `comment: TODO`
-- `content`: Object content. `content:Hello`
-- `ext`: Object extension. `ext:*.fastq.gz`
-- `handle`: Package name. `handle:examples\/metadata`
-- `hash`: Package hash. `hash:3192ac1*`
-- `key`: Object key. `key:phase*`
-- `key_text`: Analyzed object key. `key:"phase"`
-- `last_modified`: Last modified date. `last_modified:[2022-02-04 TO
-  2022-02-20]`
-- `metadata`: Package metadata. `metadata:dapi`
-- `size`: Object size in bytes. `size:>=4096`
-- `version_id`: Object version id. `version_id:t.LVVCx*`
-- `pointer_file`: Package revision tag in S3; either "latest" or a top hash. `pointer_file: latest`
-- `package_stats.total_files`: Package total files.
-  `package_stats.total_files:>100`
-- `package_stats.total_bytes`: Package total bytes.
-  `package_stats.total_bytes:<100`
+| Keyword | Description | Example |
+|- | - | - |
+| `comment`| Package comment | `comment:TODO` |
+| `content`| Object content | `content:Hello` |
+| `ext`| Object extension | `ext:*.fastq.gz` |
+| `handle`| Package name | `handle:examples\/metadata` |
+| `hash`| Package hash | `hash:3192ac1*` |
+| `key`| Object key | `key:phase*` |
+| `key_text`| Analyzed object key | `key:"phase"` |
+| `last_modified`| Last modified date | `last_modified:[2022-02-04 TO 2022-02-20]`|
+| `metadata` | Package metadata | `metadata:dapi` |
+| `size` | Object size in bytes | `size:>=4096` |
+| `version_id` | Object version id | `version_id:t.LVVCx*` |
+| `pointer_file` | Package revision tag in S3; either "latest" or a top hash | `pointer_file:latest` |
+| `package_stats.total_files` | Package total files | `package_stats.total_files:>100` |
+| `package_stats.total_bytes` | Package total bytes | `package_stats.total_bytes:<100` |
 
 **Logical operators and grouping**
 
-- `AND`: Conjunction. `a AND b`
-- `OR`: Disjunction. `a OR b`
-- `NOT`: Negation. `NOT a`
-- `_exists_`: Matches any non-null value for the given field. `_exists_: content`
-- `()`: Group terms. `(a AND b) NOT c`
+| Keyword | Description | Example |
+|- | - | - |
+| `AND` | Conjunction | `a AND b` |
+| `OR` | Disjunction | `a OR b` |
+| `NOT` | Negation | `NOT a` |
+| `_exists_` | Matches any non-null value for the given field | `_exists_: content` |
+| `()` | Group terms | `(a AND b) NOT c` |
 
 **Wildcard and regular expressions**
 
-- `*`: Zero or more characters, avoid leading `*` (slows performance).
-  `ext:config.y*ml`
-- `?`: Exactly one character. `ext:React.?sx`
-- `//`: Regular expression (slows performance). `content:/lmnb[12]/`
+| Keyword | Description | Example |
```

Comment:
these are not "keywords". i think we should keep these lists as they are

Reply:
Instead of "Keyword" we can just say "Syntax"